example/ml_workflow.py

```diff
@@ -15,15 +15,26 @@
 builder = bn.FlowBuilder('ml_workflow')
 
 # Define some basic parameters.
-builder.assign('random_seed', 0)
-builder.assign('test_split_fraction', 0.3)
-builder.assign('hyperparams_dict', {'C': 1})
-builder.assign('feature_inclusion_regex', '.*')
+builder.assign(
+    'random_seed', 0,
+    docstring='Arbitrary seed for all random decisions in the flow.')
```

Comment:
I think `doc` instead of `docstring` is fine if that's how you'd like to go!

Reply:
Done!